src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerPaymentMade.java

```diff
@@ -36,7 +37,14 @@ public class CaseworkerPaymentMade implements CCDConfig<CaseData, State, UserRol
     public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         new PageBuilder(configBuilder
             .event(CASEWORKER_PAYMENT_MADE)
-            .forStates(AwaitingHWFPartPayment, AwaitingPayment, AwaitingHWFDecision, AwaitingDocuments, AwaitingHWFEvidence)
+            .forStates(
+                AwaitingHWFPartPayment,
+                AwaitingPayment,
+                AwaitingHWFDecision,
+                AwaitingDocuments,
+                AwaitingHWFEvidence,
+                AwaitingResponseToHWFDecision
+            )
             .name("Payment made")
             .description("Payment made")
             .aboutToSubmitCallback(this::aboutToSubmit)
```

Comment:
ACs mention Awaiting applicant 
Request for information as should be showing, we'll need to track for Request for information separate. Maybe another JIRA for that so it doesn't stop this?

Reply:
oh forgot post submissions states was a complement so that fine then